day3/slice_time_image.py

```diff
@@ -87,9 +137,25 @@ def main():
     # slice is index 0 and 0 is an even number.  What are the slice acquisition
     # times in seconds, where the first value is the acquisition time of slice
     # 0, the second is acquisition time of slice 1, etc?
-    slice_times = ?
-    slice_time_file(fname, slice_times, TR)
 
+    interval = TR / n_slices # acquisition time interval in seconds between each consecutive slice
+    # Identify sequence of slices per TR
+    even_slices = []  
+    odd_slices = []
+    for islice in range(n_slices):
+        if islice % 2 == 0:
+            even_slices = even_slices + [islice]
+        else: 
+            odd_slices = odd_slices + [islice]
+    slice_seq = even_slices + odd_slices
+    lag = 0    
+    slice_lag = np.zeros(n_slices)
+    for it in slice_seq:
```

Comment:
I like it! 

You don't really need `slice_lag, could just use `slice_times` here.